epam-vote-system-db/src/main/resources/liquibase.properties

```diff
@@ -0,0 +1,5 @@
+driver=org.postgresql.Driver
```

Comment:
I propose move it file to db-module to be able start liquibase script without entire project